### PR TITLE
Add redis template and save invitaion token in redis

### PR DESCRIPTION
**Pull request by vishalgupta921**
Related: #9

### CODE REVIEW

File: src/main/java/com/seller/panel/config/RedisConfig.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import org.springframework.data.redis.connection.jedis.JedisConnectionFactory;
+import org.springframework.data.redis.core.RedisTemplate;
+
+@Configuration
+public class RedisConfig {
+
+    @Autowired
+    private Environment env;
+
+    @Bean
```
Comment (novicedev7291) [inline src/main/java/com/seller/panel/config/RedisConfig.java]: Does this need to be exposed as Bean ?

Comment (vishalgupta921) [reply src/main/java/com/seller/panel/config/RedisConfig.java]: not needed

Comment (novicedev7291) [reply src/main/java/com/seller/panel/config/RedisConfig.java]: Please remove it.

Comment (vishalgupta921) [reply src/main/java/com/seller/panel/config/RedisConfig.java]: done

**Pull request merged**